docs/source/examples/workspaces/openstack-ocp/OCP.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Don't commit the private key

Reply:
I'll delete this later, but for the current test, I'll keep it at this moment